mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb

```diff
@@ -274,6 +274,156 @@ func TestAccComputeSecurityPolicy_withRateLimitWithRedirectOptions(t *testing.T)
 	})
 }
 
+func TestAccComputeSecurityPolicy_withRecaptchaOptionsConfig(t *testing.T) {
+	t.Parallel()
+
+	project := getTestProjectFromEnv()
+	spName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	keyDisplayName := fmt.Sprintf("tf-test-display-name-%s", randString(t, 10))
+	keyDisplayNameUpdate := fmt.Sprintf("tf-test-display-name-update-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck: func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		CheckDestroy: testAccCheckComputeSecurityPolicyDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeSecurityPolicy_basic(spName),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withRecaptchaOptionsConfig(project, spName, keyDisplayName),
+			},
+			{
+				ResourceName:      "google_compute_security_policy.policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeSecurityPolicy_withRecaptchaOptionsConfig(project, spName, keyDisplayNameUpdate),
```

Comment:
it looks like `keyDisplayName` is only used in `google_recaptcha_enterprise_key`, which isn't the resource under test - why do we need this configuration change?